packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -51,11 +51,6 @@ const GlobalHeader = ({ className }) => {
   const [searchQuery, setSearchQuery] = useState('');
   const { t } = useThemeTranslation();
 
-  const handleSignupClick = useInstrumentedHandler(null, {
-    actionName: 'signup_click',
```

Comment:
One more question while I'm here. Do we not want to instrument `signup_click` through NR browser anymore?

Reply:
no we do not! that was not part of the plan - it was always supposed to be sent through tessen

Reply:
also the `signup_clicks` are actually meant to be named `stitchedDocPath` and by using the link component we are covering this case already